pyemma/coordinates/clustering/regspace.py

```diff
@@ -149,21 +145,26 @@ def _estimate(self, iterable, **kwargs):
                     regspace.partial_fit(X.astype(np.float32, order='C', copy=False), n_jobs=self.n_jobs)
                     used_frames += len(X)
             self._converged = True
-        except regspace.MaxCentersReachedException:
-            self._converged = False
-            msg = 'Maximum number of cluster centers reached.' \
-                  ' Consider increasing max_centers or choose' \
-                  ' a larger minimum distance, dmin.'
-            self.logger.warning(msg)
-            warnings.warn(msg)
-            # pass amount of processed data
-            used_data = used_frames / float(it.n_frames_total()) * 100.0
-            raise NotConvergedWarning("Used data for centers: %.2f%%" % used_data)
+        except Exception as e:
+            if 'MaxCentersReachedException' in e.__class__.__name__:
```

Comment:
Can't this exception be caught explicitly?

Reply:
with the next deeptime release yes :) right now it's set up a bit unfortunate with the rmsd metric so i can only catch it implicitly.